bin/node/cli/src/chain_spec.rs

```diff
@@ -284,7 +284,7 @@ pub fn testnet_genesis(
 			}).collect::<Vec<_>>(),
 		},
 		pallet_staking: StakingConfig {
-			validator_count: initial_authorities.len() as u32 * 2,
+			validator_count: initial_authorities.len() as u32,
```

Comment:
offchain elections are only acceptable if the provide _exactly enough_ validators. So since we have double the `initial_authorities`, offchain elections always fail on a dev chain and it produces some warn/erorr logs. 

can revert if it offends anything else.